apps/condo/domains/notification/README.md

```
@@ -68,3 +68,47 @@ TESTS_FAKE_CLIENT_MODE=true
+
+## How to add a new provider
+
+1. Implement an adapter class with static `type` and methods: `isConfigured`, `isEmailSupported`, `checkIsAvailable`, `send`
+2. Register it in the `EMAIL_ADAPTERS` map in `emailAdapter.js`
+3. Set `EMAIL_API_CONFIG.type` to that type value
+
+## Mailgun (default)
+
+Required: `api_url`, `token`, `from`.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)